components/analytics/table_chart.tsx

```diff
@@ -1,33 +1,23 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {OverlayTrigger, Tooltip} from 'react-bootstrap';
 
 import Constants from 'utils/constants';
 
-export default class TableChart extends React.PureComponent {
```

Comment:
Looks like this is from your other PR: https://github.com/mattermost/mattermost-webapp/pull/3858
Can you remove these changes?